.github/workflows/ci.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         run: clang --version
 
       - name: Install Homebrew packages
-        run: brew install boost libevent qt@5 miniupnpc libnatpmp ccache zeromq qrencode libtool automake gnu-getopt
+        run: brew install automake libtool pkg-config gnu-getopt ccache boost libevent miniupnpc libnatpmp zeromq qt@5 qrencode
```

Comment:
nit: not sure if you meant to reorder the packages for this diff, but would be a good chance to order alphabetically or by size of package

Reply:
The new order is as follows: essential build tools, optional build tools, essential dependencies, optional dependencies.